app/models/partnership_csv_upload.rb

```diff
@@ -47,6 +47,7 @@ def find_school_errors
       school = School.includes(:partnerships).find_by(urn: urn)
       school = School.find_by(urn: urn.sub!(/^[0]+/, "")) if school.blank?
 
+      # TODO: Should we dependent on `Cohort.current` here or should this model reference cohort it was used for?
```

Comment:
let's change it to be the cohort on the form

Reply:
Done!